test/unit/foreman/renderer/snapshots/ProvisioningTemplate/PXELinux/PXELinux default local boot.snap.txt

```diff
@@ -1,6 +1,3 @@
-
-
-
```

Comment:
This removal of useless whitespace makes me happy :)

Reply:
Yeah we have many of those, one day Foreman will generate clean kickstarts.